### DISCUSSION TITLE
Welcome to pldotnet Discussions!

### DISCUSSION THREAD

**Discussion by tlewis-ba** [Brick-Abode/pldotnet#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with pl/dotnet users. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

You also can email us at [pldotnet@brickabode.com](mailto:pldotnet@brickabode.com).

**Comment by ewinnington** [comment]
First of all, congratulations on the first candidate release! While I don't have a use case right now, I'll be sure to follow along and find a case for embedded dotnet on postgresql in the future!

Wonder if a the minimal Embeddings and Embeddings search (Cosine similarity) I wrote in csharp can be used on db level instead of PgVector to calculate similarity?

**Reply by tlewis-ba** [reply]
Eric, thanks for your query; you're the first poster here!

I for one would be thrilled for you to try your function as a c# stored procedure.  Start a new topic, keep us updated on your progress, and ping us with any questions; we would love to help.  I hope that it would be competitive with or faster than pgvector if we can get the JIT really cooking on it.
